ast/dml.go

```diff
@@ -2300,3 +2301,61 @@ func (n *FrameBound) Accept(v Visitor) (Node, bool) {
 	}
 	return v.Leave(n)
 }
+
+type SplitIndexRegionStmt struct {
+	dmlNode
+
+	Table      *TableName
+	IndexName  string
+	ValueLists [][]ExprNode
+}
+
+func (n *SplitIndexRegionStmt) Restore(ctx *RestoreCtx) error {
+	ctx.WriteKeyWord("SPLIT TABLE ")
+	if err := n.Table.Restore(ctx); err != nil {
+		return errors.Annotate(err, "An error occurred while restore SplitIndexRegionStmt.Table")
+	}
+	ctx.WriteKeyWord(" INDEX ")
+	ctx.WriteName(n.IndexName)
+	ctx.WriteKeyWord(" BY ")
+	for i, row := range n.ValueLists {
+		if i != 0 {
+			ctx.WritePlain(",")
```

Comment:
It seems that the last row will add the `,` too， is it OK？
Please add tests for this function.

Reply:
No, `,` will be added before `(value)`, and there is already a test in https://github.com/pingcap/parser/pull/297/files#diff-688912c3f38a8a80d6bdc16c02088d69R641

Reply:
Got it.